nexus/db-queries/src/db/queries/next_item.rs

```diff
@@ -81,23 +82,24 @@ use uuid::Uuid;
 /// FROM
 ///     (
 ///         SELECT
-///             shift
+///             index, shift
 ///         FROM
-///             generate_series(0, <max_shift>)
-///         AS shift
+///             generate_series(0, <max_shift>) AS shift,
+///             generate_series(0, <n>) AS index
```

Comment:
turbo doc nit: if I play around with this query in a CockroachDB cluster I have to quote `"index"` in order to get the query to work; presumably Diesel is doing this for us, but it's nice to be able to muck around with the query by hand to understand how it works